helm/charts/neo4j-vkaci/values.yaml

```diff
@@ -214,7 +214,7 @@ config:
 # securityContext defines privilege and access control settings for a Pod or Container. Making sure that we dont run Neo4j as root user.
 securityContext:
   runAsNonRoot: true
-  runAsUser: 1000700000
```

Comment:
is this change required

Reply:
I will revert this. This value is different for each setup